docs/source/ecf.rst

```diff
@@ -1274,7 +1274,8 @@ This file describes the transit/eclipse and systematics parameters and their pri
 Available fitting parameters are:
 
    - Transit and Eclipse Depth Parameters
-      - ``rp`` or ``rprs`` - planet-to-star radius ratio, for the transit models.
+      - ``rp`` or ``rprs`` - planet-to-star radius ratio, for most transit models.
+      - ``a0`` - planet-to-star radius ratio for Harmonica.  Add  ``a1`` to fit different morning/evening limbs.  Add ``b1`` to fit different north/south limbs.  Higher-order harmonics (up to ``n=3``) are also available.
```

Comment:
We should add a link to the harmonica docs here, since these instructions aren't thorough enough to fully understand all the parameters and their nuances (e.g., I can't tell what units `a1` and `b1` have)